tests/e2e/tslint.json

```diff
@@ -121,7 +116,8 @@
       "check-decl",
       "check-operator",
       "check-separator",
-      "check-type"
+      "check-type",
+      "check-module"
     ]
   }
 }
```

Comment:
missing new line at the end of file.

Is this also a linter rule that can be enforced?